ahoy.js

```diff
@@ -359,13 +360,25 @@
   };
 
   ahoy.trackAll = function() {
+    ahoy.start();
     ahoy.trackView();
     ahoy.trackClicks();
     ahoy.trackSubmits();
     ahoy.trackChanges();
   };
 
-  $(createVisit);
+  ahoy.start = function (options) {
+    ahoy.configure(options || {});
+    createVisit();
+
+    for (var i = 0; i < eventQueue.length; i++) {
+      trackEvent(eventQueue[i]);
+    }
+
+    ahoy.start = function () {};
+  };
+
+  if (config.startOnReady) { $(function () { ahoy.start(); }); }
```

Comment:
I'm gonna change it so that start doesn't accept the options hash; it's not necessary and just makes it more complicated.